webapp/channels/src/actions/invite_actions.ts

```diff
@@ -280,14 +281,14 @@ export function sendMembersInvitesToChannels(
                 response = {
                     data: emails.map((email) => ({
                         email,
-                        error: {error: localizeMessage('invite.members.unable-to-add-the-user-to-the-team', 'Unable to add the user to the team.')},
+                        error: {error: defineMessage({id: 'invite.members.unable-to-add-the-user-to-the-team', defaultMessage: 'Unable to add the user to the team.'})},
```

Comment:
You have several `defineMessage` written in one single line. Not sure if intended or not, but I like them better in several lines (one for id, one for default message). You do both in this file, so consistency would also be nice :stuck_out_tongue_winking_eye:

Reply:
Yeah, I prefer having these split onto multiple lines as well. I left it as-is for the sake of keeping diff more readable, but I don't think splitting those onto multiple lines will make that worse, so I'll change that now